tensorflow/core/profiler/g3doc/command_line.md

```diff
@@ -57,7 +57,7 @@ Note: this feature is not well maintained now.
 
 ```shell
 # Build the tool.
-bazel build --config opt third_party/tensorflow/core/profiler/...
+bazel build --config opt tensorflow/core/profiler:profiler
```

Comment:
Leave the ... part?

Reply:
Not sure about what the `...` mean, maybe should be kept.